Setup/install.sh

```diff
@@ -283,8 +244,6 @@ install_wiringpi
 install_spi_i2c
 # no longer installing avr for arduino
 # install_avr
-install_line_follower
-install_control_panel
```

Comment:
why is the control panel removed?

Reply:
Good call. I knew something was missing. Thanks.